pkg/features/README.md

```diff
@@ -6,30 +6,41 @@ Currently, feature flags use environment variables to power the toggles.
 
 Feature flags can be valuable to ship features in a preview state, to provide the end user a way to disable some functionality or to control any boolean setting.
 
-> :warning: Feature flags cannot be used inside migrations or schema changes.
-> Migrations must be merged to the master branch without any feature flag gate, and must not break any current features.
+## Use cases
 
-## Adding a feature flag
+### Feature release stages
 
-To add a feature flag, add a variable with your feature to `list.go`. To register this feature flag variables, you are required to provide the following:
+To promote a feature through the release stages, consider the following progression:
 
-* Name: This is a short description of what this flag is about
-* Environment Variable name: This is the environment variable which needs to be set to override the default value of this flag. Env var names **must** start with `ROX_`
-* Options:
-  * whether the flag is changeable on release builds (default) or not (add `unchangeableInProd` option). See below for more details
-  * default value: disabled (default) or enabled (add `enabled` option)
+1. Until the feature is ready to be tested by customers, the flag should be disabled by default and unchangeable in production.
+2. The flag, disabled by default, can be made changeable to allow customers share feedback on the feature.
+3. When the feature is complete and generally available, the flag can be enabled by default.
+4. After the feature has been tested in production for some time, but not longer, remove the flag.
```

Comment:
nit: wording "tested in production" feels contradictory for me. How about "active in production" instead.

Reply:
This is a punch to the idea of having a flag enabled by default. Thanks for noticing!

Reply:
FTR, the time the flag is enabled in production is the time when we look at how the feature is working before removing the flag with a raised hand over the flag deactivation button. Kinda testing.